packages/pyright-internal/src/tests/fourslash/completions.importInterimFile.fourslash.ts

```diff
@@ -34,7 +34,7 @@
         },
     });
 
-    helper.replace(0, 9, 'import altair as alt\n\nalt.');
+    helper.replace(helper.BOF, helper.getPosition('marker1'), 'import altair as alt\n\nalt.');
```

Comment:
but how does it know which file? it always assumes active file? since there can be multiple open file? is the last file always active file?

Reply:
Yes, currently it always targets the active file, which by default is the first file. That's how `replace` worked when I found it, and I didn't change that. Apparently this is the first test that has needed to edit files, so adding more flexibility seems unnecessary at the moment.